authmanager/authmanager.go

```diff
@@ -110,3 +121,12 @@ func extractBaseURL(origin string) string {
 
 	return fmt.Sprintf("%s//%s", tokens[0], tokens[1])
 }
+
+func showAuthWarning(cfg *config.Local) {
+	switch cfg.IssueTracker {
+	case config.IssueTrackerGithub:
+		fmt.Println(color.RedString(githubAPITokenWarning))
```

Comment:
This is fine, but please use a yellow color for the warning instead of red. We use red for errors and reusing it for warnings is confusing.